pom.xml

```diff
@@ -215,7 +215,7 @@
         <microprofile-healthcheck.version>4.0</microprofile-healthcheck.version>
         <microprofile-metrics.version>4.0</microprofile-metrics.version>
         <microprofile-rest-client.version>3.0</microprofile-rest-client.version>
-        <microprofile-openapi.version>3.0</microprofile-openapi.version>
+        <microprofile-openapi.version>3.1</microprofile-openapi.version>
```

Comment:
?

Reply:
I forgot to remove this, it is not needed